kyuubi-server/src/main/antlr4/org/apache/kyuubi/sql/KyuubiTrinoFeBaseParser.g4

```diff
@@ -17,50 +17,46 @@
-    | SELECT TABLE_TYPE FROM SYSTEM_JDBC_TABLE_TYPES ORDER BY TABLE_TYPE                            #getTableTypes
+      (WHERE (TABLE_CATALOG EQ catalog=STRING)? AND? (TABLE_SCHEM LIKE schema=STRING)?)?
+      ORDER BY TABLE_CATALOG COMMA TABLE_SCHEM                                                          #getSchemas
+    | SELECT TABLE_CAT FROM SYSTEM_JDBC_CATALOGS ORDER BY TABLE_CAT                                     #getCatalogs
+    | SELECT TABLE_TYPE FROM SYSTEM_JDBC_TABLE_TYPES ORDER BY TABLE_TYPE                                #getTableTypes
     | SELECT TYPE_NAME COMMA DATA_TYPE COMMA PRECISION COMMA LITERAL_PREFIX COMMA
       LITERAL_SUFFIX COMMA CREATE_PARAMS COMMA NULLABLE COMMA CASE_SENSITIVE COMMA
       SEARCHABLE COMMA UNSIGNED_ATTRIBUTE COMMA FIXED_PREC_SCALE COMMA AUTO_INCREMENT
       COMMA LOCAL_TYPE_NAME COMMA MINIMUM_SCALE COMMA MAXIMUM_SCALE COMMA SQL_DATA_TYPE
-      COMMA SQL_DATETIME_SUB COMMA NUM_PREC_RADIX FROM SYSTEM_JDBC_TYPES ORDER BY DATA_TYPE         #getTypeInfo
+      COMMA SQL_DATETIME_SUB COMMA NUM_PREC_RADIX FROM SYSTEM_JDBC_TYPES ORDER BY DATA_TYPE             #getTypeInfo
     | SELECT TABLE_CAT COMMA TABLE_SCHEM COMMA TABLE_NAME COMMA TABLE_TYPE COMMA REMARKS COMMA
       TYPE_CAT COMMA TYPE_SCHEM COMMA TYPE_NAME COMMA SELF_REFERENCING_COL_NAME COMMA REF_GENERATION
       FROM SYSTEM_JDBC_TABLES
       (WHERE tableCatalogFilter? AND? tableSchemaFilter? AND? tableNameFilter? AND? tableTypeFilter?)?
-      ORDER BY TABLE_TYPE COMMA TABLE_CAT COMMA TABLE_SCHEM COMMA TABLE_NAME                        #getTables
-    | .*?                                                                                           #passThrough
+      ORDER BY TABLE_TYPE COMMA TABLE_CAT COMMA TABLE_SCHEM COMMA TABLE_NAME                            #getTables
+    | .*?                                                                                               #passThrough
     ;
 
 tableCatalogFilter
-    : TABLE_CAT IS NULL                                                                             #nullCatalog
-    | TABLE_CAT EQ catalog=STRING+                                                                  #catalogFilter
+    : TABLE_CAT IS NULL                                                                                 #nullCatalog
+    | TABLE_CAT EQ catalog=STRING                                                                       #catalogFilter
     ;
 
 tableSchemaFilter
-    : TABLE_SCHEM IS NULL                                                                           #nulTableSchema
-    | TABLE_SCHEM LIKE schemaPattern=STRING+ ESCAPE SEARCH_STRING_ESCAPE                            #schemaFilter
+    : TABLE_SCHEM IS NULL                                                                               #nulTableSchema
+    | TABLE_SCHEM LIKE schemaPattern=STRING ESCAPE STRING                                               #schemaFilter
```

Comment:
emm, It should be `ESCAPE '\'` instead of `ESCAPE STRING`, `ESCAPE '\'` is fixed.

Like `... ESCAPE '\'` can be parsed to `GetSchemas`, but `... ESCAPE 'aaa'` can not.